go.mod

```diff
@@ -5,7 +5,7 @@ go 1.13
 require (
 	github.com/MakeNowJust/heredoc v1.0.0
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/coredns/corefile-migration v1.0.6
+	github.com/coredns/corefile-migration v1.0.7-0.20200317205912-4b5f6e751512
```

Comment:
We should probably stick with the latest released version, is there any reason this was updated to a commit?

Reply:
@rajansandeep are you folk planning to release 1.0.7 soon-ish?

Reply:
Yeah I was wondering the same. I updated it to the commit that exported `ValidUpMigration` with the intent for it to be temporary.

Reply:
@vincepri We usually have a release every upstream k8s cycle, but if it's necessary we can make a release for this case

Reply:
It'd be great to have a tag, if possible, so we don't have to use the hash commit here

Reply:
Release 1.0.7 is now available 
https://github.com/coredns/corefile-migration/releases/tag/v1.0.7

Reply:
🎉 thank you!! @gab-satchi over to you!

Reply:
Appreciate it @rajansandeep. Just updated @vincepri